packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -221,7 +309,14 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "closeButtonTooltip": r"Schließen",
     "nextMonthTooltip": r"Nächster Monat",
     "previousMonthTooltip": r"Vorheriger Monat",
+    "nextPageTooltip": r"Nächste Seite",
+    "previousPageTooltip": r"Vorherige Seite",
+    "showMenuTooltip": r"Zeige das Menü",
     "licensesPageTitle": r"Lizenzen",
+    "pageRowsInfoTitle": r"$firstRow–$lastRow von $rowCount",
+    "pageRowsInfoTitleApproximate": r"$firstRow–$lastRow von etwa $rowCount",
+    "rowsPerPageTitle": r"Zeilen pro Seite:",
+    "selectedRowCountTitleOther": r"$selectedRowCount Artikel ausgewählt",
```

Comment:
"Artikel" is pretty domain specific. I don't know the context of this translation, but unless it deals with things to buy or (printed) articles, you might want to consider using the more generic "Objekt" or "Element", which are closer to the generic meaning of "Item", I think.

Keine Objekte/Elemente ausgewält.
1 Objekt/Element ausgewält.
2 Objekte/Elemente ausgewält.

I'd probably go with Objekt.